From 22d625b7086d25efe5de72e2b80b9b8e98573553 Mon Sep 17 00:00:00 2001
From: Maxim Uvarov <maxim.uvarov@linaro.org>
Date: Fri, 17 Apr 2020 12:05:53 +0100
Subject: [PATCH 2/3] add enum to ta flags

If we compile this TA into OPTEE-OS we need to define a flag
that this TA can be discovered on the optee bus.
Upstream-Status: Submitted [https://github.com/microsoft/MSRSec/pull/34]

Signed-off-by: Maxim Uvarov <maxim.uvarov@linaro.org>
---
 TAs/optee_ta/fTPM/user_ta_header_defines.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/TAs/optee_ta/fTPM/user_ta_header_defines.h b/TAs/optee_ta/fTPM/user_ta_header_defines.h
index 6ff62d1..685b54a 100644
--- a/TAs/optee_ta/fTPM/user_ta_header_defines.h
+++ b/TAs/optee_ta/fTPM/user_ta_header_defines.h
@@ -44,7 +44,7 @@
 
 #define TA_UUID                     TA_FTPM_UUID
 
-#define TA_FLAGS                    (TA_FLAG_SINGLE_INSTANCE | TA_FLAG_INSTANCE_KEEP_ALIVE )
+#define TA_FLAGS                    (TA_FLAG_SINGLE_INSTANCE | TA_FLAG_INSTANCE_KEEP_ALIVE | TA_FLAG_DEVICE_ENUM_SUPP)
 #define TA_STACK_SIZE               (64 * 1024)
 #define TA_DATA_SIZE                (64 * 1024)
 
-- 
2.43.0

